.env.development

```diff
@@ -14,7 +14,7 @@ PGDATABASE=postgres
 
 SECURE_COOKIES=false
 TRUST_PROXY=false
-LIGHTDASH_SECRET='not very secret'
```

Comment:
Changing these to double quotes so after copying it's straightforward to source in a shell `profile` file.